db/schema.rb

```diff
@@ -1562,6 +1562,7 @@
     t.string "program", null: false
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.index ["program", "user_id"], name: "index_raffles_on_program_and_user_id", unique: true
```

Comment:
This was removed in https://github.com/hackclub/hcb/pull/10830/files#diff-cbf8b25d6d853acf58fa9057841f407d29ffe90649c75cf9e3f51b2d6e3aa1d3L1520 but is present (and necessary) in production:

<img width="1171" height="506" alt="CleanShot 2025-07-22 at 11 11 02@2x" src="https://github.com/user-attachments/assets/ba0fc2ab-6570-4501-abef-8fcfa6fc4551" />